Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -163,7 +166,12 @@ protected override void OnApplyTemplate()
             base.OnApplyTemplate();
         }
 
-        private void MenuItem_LayoutUpdated(object sender, object e)
+        private void MenuFlyout_Opening(object sender, object e)
+        {
```

Comment:
Is this blank method to override a default behavior or something?  If so, can you add a comment?